en-GB/docs/find.ndb

```diff
@@ -385,6 +388,7 @@ STR.nrmtch(STR,POS)~Searches for the last occurrence of a char in the string up
 STR.rfnd(STR,POS)~Returns the position of the last occurrence of STR in the string, with the search starting from the position POS~methode strings zeichenketten find last rfnd()~.rfnd(
 STR.rmtch(STR,POS)~Searches for the last occurrence of a char of STR in the string up to position POS~methode strings zeichenketten find last rmtch()~.rmtch(
 STR.splt(CHAR,KEEP)~Splits the string at the characters CHAR in separate strings~methode strings zeichenketten unterteilen tokenizer zerlegen splt()~.splt(
+STR.startsw(STR)~Gives the value True back, when the second Substring is found at the start of the first string. Otherwise, False.~begin end last erstes letztes wort word front back~.startsw(

```

Comment:
```suggestion
STR.startsw(STR)~Returns true, if the second substring is found at the start of the first string. Otherwise, false.~begin end last erstes letztes wort word front back~.startsw(
```